### PR TITLE
Corrected the changes

### PR DESCRIPTION
**Pull request by Shreya1304**
Fixes #

-head tag added
-radio buttons grouped together
-id added to the form tag
-required attribute added to the password tag

-
-
-

### CODE REVIEW

File: Shreya/HTML Basics Remake/form.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<html>
```
Comment (aravindvnair99) [inline Shreya/HTML Basics Remake/form.html:2]: Missing `DOCTYPE`.

File: Shreya/HTML Basics Remake/form.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                <input type="text" name="last" placeholder="Last Name" required>
+            </p>
+            <p>
+                <label for="gender">Gender:</label>
+              <label for="gender">Male:</label>
+                <input type="radio" name="gender" required>
+                <label for="gender">Female:</label>
+                <input type="radio" name="gender" required>
+                <label for="gender">Others:</label>
+                <input type="radio" name="gender" required>
```
Comment (aravindvnair99) [inline Shreya/HTML Basics Remake/form.html:23]: Label is for `id` and not `name`. Missing `id` attribute.

File: Shreya/HTML Basics Remake/form.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+                <label for="email">Email:</label>
+                <input type="email" placeholder="Email">
+                <label for="password">Password:</label>
+                <input type="password" pattern=".{5,15}" title="PLease enter minimum 5 and maximum 15 characters" required>
+                <label for="date">Birthday</label>
+            <input type="date">
+            </p>
+            <p>
+                <label for="check">I agree to the terms and conditions</label>
+                <input type="checkbox" name="check">
```
Comment (aravindvnair99) [inline Shreya/HTML Basics Remake/form.html]: Missing `required` attribute.

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

File: Shreya/HTML Basics Remake/form.html [hunk 4]
```diff
@@ -8,22 +9,22 @@
         <h1>Register</h1>
             <p>
                 <label for="first">First Name</label>
-                <input type="text" name="first" placeholder="First Name" required>
+                <input type="text" id="first" placeholder="First Name" required>
                 <label for="last">Last Name</label>
-                <input type="text" name="last" placeholder="Last Name" required>
+                <input type="text" id="last" placeholder="Last Name" required>
```
Comment (aravindvnair99) [inline Shreya/HTML Basics Remake/form.html]: Why change from `name` to `id`? Revert it back to `name`

Refer this: [What is the difference between id and name attributes in form elements? - Treehouse](https://teamtreehouse.com/community/what-is-the-difference-between-id-and-name-attributes-in-form-elements)

File: Shreya/HTML Basics Remake/form.html [hunk 5]
```diff
@@ -8,22 +9,22 @@
...
             </p>
             <p>
-                <label for="gender">Gender:</label>
-              <label for="gender">Male:</label>
-                <input type="radio" name="gender" required>
-                <label for="gender">Female:</label>
-                <input type="radio" name="gender" required>
-                <label for="gender">Others:</label>
-                <input type="radio" name="gender" required>
+                <label for="1">Gender:</label>
```
Comment (aravindvnair99) [inline Shreya/HTML Basics Remake/form.html]: Label not required here.

File: Shreya/HTML Basics Remake/form.html [hunk 6]
```diff
@@ -8,22 +9,22 @@
...
+              <label for="2">Male:</label>
+                <input type="radio" id="2" name="gender" required>
+                <label for="3">Female:</label>
+                <input type="radio" id="3" name="gender" required>
+                <label for="4">Others:</label>
+                <input type="radio"  id="4" name="gender" required>
```
Comment (aravindvnair99) [inline Shreya/HTML Basics Remake/form.html]: Use relevant IDs and `value` is required.

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

File: Shreya/HTML Basics Remake/form.html (hunk 3 above)
Comment (aravindvnair99) [reply Shreya/HTML Basics Remake/form.html]: @Shreya1304 This still needs fixing.

**Comment by Shreya1304** [comment]
Corrected the requested changes. Please let me know if you received the
pull request.

File: Shreya/HTML Basics Remake/form.html (hunk 4 above)
Comment (Shreya1304) [reply Shreya/HTML Basics Remake/form.html]: okay

File: Shreya/HTML Basics Remake/form.html (hunk 5 above)
Comment (Shreya1304) [reply Shreya/HTML Basics Remake/form.html]: then what tag shall I wrap Gender in ?

Comment (aravindvnair99) [reply Shreya/HTML Basics Remake/form.html]: > then what tag shall I wrap Gender in ?

Refer this: [What is the HTML for=“” attribute in \<label>? - Stackoverflow](https://stackoverflow.com/a/45307833/7023294)

File: Shreya/HTML Basics Remake/form.html (hunk 3 above)
Comment (Shreya1304) [reply Shreya/HTML Basics Remake/form.html]: done

Comment (aravindvnair99) [reply Shreya/HTML Basics Remake/form.html]: > done

Checkbox and date also need it.

File: Shreya/HTML Basics Remake/form.html [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+            </p>
+            <p>
+                <label for="email">Email:</label>
+                <input type="email" placeholder="Email" required>
+                <label for="password">Password:</label>
+                <input type="password" pattern=".{5,15}" title="PLease enter minimum 5 and maximum 15 characters" required>
+                <label for="date">Birthday</label>
+            <input type="date">
```
Comment (aravindvnair99) [inline Shreya/HTML Basics Remake/form.html:31]: Missing autocomplete. Refer this: [The HTML autocomplete attribute - MDN web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete)

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

**Comment by aravindvnair99** [comment]
@Shreya1304 Updates? It's been 22 days.

File: Shreya/HTML Basics Remake/form.html (hunk 5 above)
Comment (VinayakTekade) [reply Shreya/HTML Basics Remake/form.html]: Still incomplete

File: Shreya/HTML Basics Remake/form.html (hunk 7 above)
Comment (VinayakTekade) [reply Shreya/HTML Basics Remake/form.html:31]: Still incomplete

**Review by VinayakTekade — DISMISSED** [review]

File: Shreya/HTML Basics Remake/form.html (hunk 7 above)
Comment (Shreya1304) [reply Shreya/HTML Basics Remake/form.html:31]: @aravindvnair99  you want me to put autocomplete in al the input elements?

**Review by aravindvnair99 — COMMENTED** [review]
@Shreya1304 Merging and closing the pull request for now. Make the necessary changes in another pull request and send it over from a different branch. Also, make sure to fill up the pull request template and also refactor the code henceforth.

**Pull request merged**